sklearn/cluster/_k_means.pyx

```diff
@@ -163,19 +166,23 @@ def _mini_batch_update_csr(X, np.ndarray[DOUBLE, ndim=1] x_squared_norms,
          The vector in which we keep track of the numbers of elements in a
          cluster
 
+    nearest_center: array, shape (n_samples,)
+        The vector in which we keep track of the nearest center of each
+        sample
+
+    old_center: array, shape (n_features,)
+        The vector in which we keep track of the old center of each feature
+
+    compute_squared_diff: integer
```

Comment:
I read the source code for this and I didn't understand why exactly are we using this?